### ISSUE TITLE
(Pending) Credential is missing ceterms:subjectWebpage

### ISSUE THREAD

**Issue by siuc-nate** [CredentialTransparencyInitiative/vocabularies#294]
I can't find any evidence that this was removed intentionally - is this an error?

**Comment by stuartasutton** [comment]
Truly odd. It was there when I checked while developing the images. Nate,
I've not touched the spreadsheet itself in at least 6 weeks...could
something be going wrong?

**Comment by stuartasutton** [comment]
No, my response was not correct. I had already used the property and did not double check on `Credential`. It should obviously be there.

**Comment by siuc-nate** [comment]
Per our 1/31/2017 call, we will add ceterms:Credential to the domain of ceterms:subjectWebpage.
